man/systemd.service.xml

```diff
@@ -622,6 +622,12 @@
         <citerefentry><refentrytitle>sd_notify</refentrytitle><manvolnum>3</manvolnum></citerefentry>).
         </para>
 
+        <para>If the service is of type <varname>Type=notify-reload</varname> then this also configures the
+        time to wait for the reload to complete. If the service does not signal reload completion
+        (through <literal>READY=1</literal>) within the configured time, the reload will fail. This will not
+        affect the running service, but will be logged and will cause e.g. <command>systemctl reload</command>
+        to fail.</para>
```

Comment:
this is unecessarily precise. the start timeout is applied to any kind of reloads, regardless if Type=notify-reload or implemented via ExecReload=, it's all the same.

Hence a brief notice "Note that the start timeout is also applied to service reloads, regardless if implemented via <varname>ExecReload=</varname> or via the reload logic enabled via <varname>Type=notify-reload</varname>." should suffice.

Reply:
Will adjust. Thanks for the clarification